File: Baekjoon_Online_Judge/NO_05522/5522.py
# -*- encoding: utf-8 -*-
import sys
r_input = sys.stdin.readline

# ---------------------------- #
# | Created by happiness96   | #
# | Year 2020                | #
# | Month 07                 | #
# | Day 19                   | #
# | 5522 카드 게임            | #
# ---------------------------- #


if __name__ == "__main__":
    print(sum([int(r_input()) for _ in range(5)]))

File: Baekjoon_Online_Judge/NO_05522/5522.c
#include <stdio.h>
/*
 ---------------------------- 
 | Created by happiness96   | 
 | Year 2020                | 
 | Month 07                 | 
 | Day 19                   | 
 | 5522 카드 게임            | 
 ---------------------------- 
*/

int main(){
    int total = 0;

    for(int i = 0; i < 5; i++){
        int number;
        scanf("%d", &number);

        total += number;
    }

    printf("%d", total);
    return 0;
}
